# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/java/sun-jdk14/head-tail.patch
# ROCK Linux is Copyright (C) 1998 - 2006 Clifford Wolf
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- j2sdk-1_4_2_08-linux-i586.bin	2005-05-21 03:31:21.000000000 +0200
+++ java.bin	2005-05-21 11:43:00.000000000 +0200
@@ -355,7 +355,7 @@
     esac
 done
 outname=install.sfx.$$
-diskSpace=`df -k . | tail -1 | awk '{if ( $4 ~ /%/) { print $3 } else { print $4 } }'`
+diskSpace=`df -k . | tail -n1 | awk '{if ( $4 ~ /%/) { print $3 } else { print $4 } }'`
 if [ $diskSpace -lt $diskSpaceRequired ]; then
     printf "You will need atleast %s kBytes of Disk Free\n" $diskSpaceRequired
     printf "Please free up the required Disk Space and try again\n"
@@ -363,7 +363,7 @@
 fi
 trap 'rm -f $outname; exit 1' HUP INT QUIT TERM
 echo "Unpacking..."
-tail +511 $0 > $outname
+tail -n+511 $0 > $outname
 if [ -x /usr/bin/sum ] ; then
     echo "Checksumming..."
 
